### ISSUE TITLE
Error messages showed when in systemd-boot

### ISSUE THREAD

**Issue by aplanas** [kubic-project/health-checker#20]
When dracut is generated, this is presented:

dracut[I]: *** Including module: drm ***
dracut[I]: *** Including module: health-checker ***
dracut-install: ERROR: installing 'grub2-editenv'
dracut[E]: FAILED: /usr/lib/dracut/dracut-install -D /var/tmp/dracut.IEcUqm/initramfs -a date btrfs awk grub2-editenv
dracut[I]: *** Including module: pcr-signature ***

But a new initrd is generated

**Comment by laenion** [comment]
I guess this is specific to systems using sdboot instead of GRUB2.

The _module-setup.sh_ rule will have to be adjusted to ignore grub2-editenv on non-GRUB2 systems.

**Comment by thkukuk** [comment]
We need a complete new health-checker design, Danilo is on it.

Ignaz Forster ***@***.***> schrieb am Mi., 14. Aug. 2024,
17:08:

**Comment by Vogtinator** [comment]
Yeah, currently health-checker will not work to any extent with sdbootutil, not even the simplest case of rollback due to service failure (it uses `btrfs subvol set-default` instead of `snapper rollback` or `t-u rollback`).

My suggestion for now is to not install health-checker with sdbootutil or ignore the errors.
